vispy/visuals/gridlines.py

```diff
@@ -75,7 +96,9 @@ class GridLinesVisual(ImageVisual):
         channel modified.
     """
 
-    def __init__(self, scale=(1, 1), color='w'):
+    def __init__(self, scale=(1, 1), color='w',
+                 bounds=(-np.inf, np.inf, -np.inf, np.inf),
```

Comment:
This needs to be added to the docstring and the element order described (xmin, ymin, xmax, ymax)?

Reply:
I would also find it more intuitive to allow `None` as a bound as convenience, and just convert to the appropriate `inf` in the setter.

Reply:
:+1: on that idea.

Reply:
Sounds good, do you want to add to this PR for that functionality?

Reply:
Yes. The `-inf` functionality should be possible/supported, but it should probably default to 4 `None` and have those converted to `inf` as a convenience. You could even support a single `=None` and convert that to `(-np.inf, np.inf, -np.inf, np.inf)`.